src/logging/LoggingService.ts

```diff
@@ -14,77 +14,38 @@
  * limitations under the License.
  */
 
+import {Property} from '../config/Properties';
 import {DefaultLogger} from './DefaultLogger';
-import {NoLogger} from './NoLogger';
+import {ILogger} from './ILogger';
 
 export enum LogLevel {
+    OFF = -1,
```

Comment:
`OFF` is not a LogLevel. It is rather a state of logging.

Reply:
@sancar wanted me to make this change. Also, it is used as log level in other log libraries. See: https://www.tutorialspoint.com/log4j/log4j_logging_levels.htm